### DISCUSSION TITLE
[@types/websql] nothing is exported?

### DISCUSSION THREAD

**Discussion by cjhewett** [DefinitelyTyped/DefinitelyTyped#62473] [Issues with a @types package] [answered]
I am fairly new to TS, and am trying to make use of the `websql` type
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/websql/index.d.ts

However there are no `export` statements in that file so I am confused how to make use of the interfaces?

All the other types I have looked at have `export interface...` and so work as expected using an `import {x} from 'y'`

[a comment by an automated account is not shown]

**Comment by TeamworkGuy2** [comment, marked as answer]
The `websql` type definition is written for a global type. You don't import `websql` because it's globally available in the environments that support it (i.e. chrome, [for now](https://developer.chrome.com/blog/deprecating-web-sql/)) so you don't need to import this type.
The `Window` extension and `declare` types in the definition file are globally available:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/727bae0d5bc67bf017ff9746bb12c2a6a6eb8afa/types/websql/index.d.ts#L11
Which points to:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/727bae0d5bc67bf017ff9746bb12c2a6a6eb8afa/types/websql/index.d.ts#L20-L21

And a few other helper interfaces:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/727bae0d5bc67bf017ff9746bb12c2a6a6eb8afa/types/websql/index.d.ts#L166

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/727bae0d5bc67bf017ff9746bb12c2a6a6eb8afa/types/websql/index.d.ts#L181
